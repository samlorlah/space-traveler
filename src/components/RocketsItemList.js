import React from 'react';

const RocketsItemList = () => (
  <li>
    <div className="images">
      <img src="https://via.placeholder.com/200" alt="" />
    </div>
    <div className="description">
      <h4>Falcon 1</h4>
      <p>
        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
        Lorem Ipsum has been the industry&apos;s standard dummy text ever since the 1500s,
        when an unknown printer took a galley of type and scrambled it to make a type
        specimen book. It has survived not only five centuries, but also the leap into
        electronic typesetting, remaining essentially unchanged. It was popularised in
        the 1960s with the release of Letraset sheets containing Lorem Ipsum passages,
        and more recently with desktop publishing software like Aldus PageMaker including
        versions of Lorem Ipsum.
      </p>
      <button className="reserveRocketBtn" type="button">Reserve Comment</button>
    </div>
  </li>
);

export default RocketsItemList;
