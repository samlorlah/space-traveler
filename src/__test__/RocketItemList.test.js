import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import RocketsItemList from '../components/RocketsItemList';

it('Renders RocketsItems component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <RocketsItemList />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
