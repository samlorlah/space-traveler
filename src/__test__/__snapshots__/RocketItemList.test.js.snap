// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders RocketsItems component correctly 1`] = `
<li>
  <div
    className="images"
  >
    <img
      alt="undefined"
      height={150}
      width={200}
    />
  </div>
  <div
    className="description"
  >
    <h4 />
    <p>
       
    </p>
    <button
      className="reserveRocketBtn"
      onClick={[Function]}
      type="button"
    >
      Reserve Rocket
    </button>
  </div>
</li>
`;
